src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -560,6 +566,49 @@ public static K8SConfiguration LoadKubeConfig(Stream kubeconfigStream)
             return LoadKubeConfigAsync(kubeconfigStream).GetAwaiter().GetResult();
         }
 
+        /// <summary>
+        ///    Loads Kube Config from KUBECONFIG environment variable
+        /// </summary>
+        /// <param name="useRelativePaths">When <see langword="true"/>, the paths in the kubeconfig file will be considered to be relative to the directory in which the kubeconfig
+        /// <returns>Instance of the <see cref="K8SConfiguration"/> class</returns>
+        public static K8SConfiguration LoadKubeConfigFromEnvironmentVariable(string environmentVariable = "KUBECONFIG", bool useRelativePaths = true)
```

Comment:
Does this really need to be a separate method?

Reply:
I'll make this internal.